lib/utils.re

```diff
@@ -164,8 +146,94 @@ let checkAcceptable = (classNames, loc, tailwindFile) => {
   List.iter(isAcceptable, classNames);
 };
 
-let validate = (~classNames, ~loc, ~tailwindFile) => {
+exception Ppx_cache_dir_is_not_dir;
+
+// Source: https://github.com/reasonml-community/graphql_ppx/blob/master/src/base/read_schema.re#L301
+let create_dir_if_not_exist = abs_path =>
```

Comment:
I took this function from the `graphql_ppx` repo. For now errors are not handled but I guess we can improve that later on

Reply:
Can we camelCase this (`createDirIfNotExist`) and the exception? I'd just like to be consistent with the rest of the codebase.